scripts/download-import-schema-from-gecko-dev

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+const path = require('path');
+
+const shell = require('shelljs');
+
+// Fail as soon as a shelljs command fails.
+shell.config.fatal = true;
+
+const gitClone = {
```

Comment:
I'd declare it like this:

```
const GIT_CONFIG = {
```

because (1) it is some config and (2) a global variable modified in `processScriptOptions()`

Reply:
:+1: sgtm too, I will update the PR accordingly